Networking/WebSockets/WebSocketProtocol.hh

```diff
@@ -337,25 +332,28 @@ namespace uWS {
             memcpy(dst + headerLength, src, length);
 
             if ( !isServer ) {
-                // overwrites up to 3 bytes outside of the given buffer!
-                //WebSocketProtocol<isServer>::unmaskInplace(dst + headerLength, dst + headerLength + length, mask);
-
-                // this is not optimal
-                char* start = dst + headerLength;
-                char* stop  = start + length;
-                int   i     = 0;
-                while ( start != stop ) { (*start++) ^= mask[i++ % 4]; }
+                uint32_t mask_i32 = *reinterpret_cast<uint32_t*>(mask);
+
+                auto*     start = reinterpret_cast<uint32_t*>(dst + headerLength);
+                uint32_t* end   = start + length / 4;
+
+                while ( start != end ) { *start++ ^= mask_i32; }
```

Comment:
Additional logic? Is this needed?

Reply:
It seems to be addressing the comment "This is not optimal" by making it a bit more optimal.